js/app.js

```diff
@@ -9,4 +9,127 @@ hamburger.addEventListener('click', () => {
 navMenu.addEventListener('click', () => {
   hamburger.classList.toggle('active');
   navMenu.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+const ProjectCardsDetails = [
+  {
+    id: 'work1',
```

Comment:
- I know there are lots of points to modify, but there are also things that I don't understand why I was asked to change before and now have to put it back as it was, and in this case would be the button style, I couldn't use the "white" font color because the color used on the design never passed the accessibility checks, and my reviewer from that activity told me to use a color with a different contrast, so I changed it and then all the accessibility checks were fine. And now I have to put the "white" font color back? so then It will look as the design again but won't be accessible? I don't know.

 - Another thing is that my project doesn't actually use a logo, I put it in when I started to built the project and I just haven't delete that section because I think I can use it later, when there won't be a review, like 2 that I had before, that tells me to remove it because my template doesn't have one. But now, that you pointed me that "my logo disappeared", a logo that doesn't exist in the template, I don't know if I should use it or not...